bundle/Resources/config/services.yml

```diff
@@ -315,6 +317,14 @@ services:
     ezrepoforms.user_register.registration_group_loader.configurable:
         class: EzSystems\RepositoryForms\UserRegister\ConfigurableRegistrationGroupLoader
         arguments:
-            - @ezpublish.api.repository
+            - "@ezpublish.api.repository"
         calls:
             - [setGroupId, ["$users.registration_group$"]]
+
+    ezrepoforms.form_data_mapper.user_register:
+        class: EzSystems\RepositoryForms\Data\Mapper\UserRegisterMapper
+        arguments:
+            - "@ezpublish.api.repository"
```

Comment:
very good one @glye. 
it seems YAML specification doesn't allow start northing with `@`. so you MUST quote it. It happens symfony 2 yaml parser allows to do that and this is cause it's working for ez until now. 

But looks symfony 3 will follow that YAML spec so maybe we will need to quote all this kind of service arguments...